src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -90,7 +90,7 @@ public Response createUserUponOtpValidation(JsonNode payload) {
         return inviteValidator.validateOtpValidationRequest(payload)
                 .map(errors -> Response.status(BAD_REQUEST).entity(errors).build())
                 .orElseGet(() -> {
-                    User createdUser = inviteService.createInvitedUser(InviteValidateOtpRequest.from(payload));
+                    User createdUser = inviteService.createInvitedUserAndInvalidateInvite(InviteValidateOtpRequest.from(payload));
```

Comment:
Just a suggestion. While I prefer descriptive method names, my personal preference is the name to describe key intended actions rather than any side effects. So in my opinion what this service intends to do is validate the top and create the user. Disable Invite for me is a side effect rather than a key action. 
I would have named this like `validateOtpAndCreateUser`